### PR TITLE
Add toggle sort in plugin

### PR DESCRIPTION
**Pull request by webmastak**
Add toggle sort in plugin, <kbd>Ctrl</kdb>+<kdb>T</kbd> `bindkey '^T' toggle-fzf-tab`

### CODE REVIEW

**Comment by romkatv** [comment]
Did you just copy changes from https://github.com/Aloxaf/fzf-tab/pull/46?

File: fzf-tab.zsh [hunk 1]
```diff
@@ -358,15 +380,36 @@ enable-fzf-tab() {
...
+    fi
+}
+
+disable-sort-fzf-tab() {
+    typeset -g _ZSH_FZF_TAB_SORT
+    emulate -L zsh
+	zstyle ':fzf_tab:*' sort false 
+}
+
+enable-sort-fzf-tab() {
```
Comment (romkatv) [inline fzf-tab.zsh]: Adding enable/disable functions for every boolean zstyle options is not wise.

Comment (webmastak) [reply fzf-tab.zsh]: However, it works and not bad.

Comment (romkatv) [reply fzf-tab.zsh]: It is bad as it creates unnecessary API bloat. Every public symbol has to be documented, read by users and supported forever.

Comment (webmastak) [reply fzf-tab.zsh]: Can you make the sorting correct?

**Comment by Aloxaf** [comment]
1. Don't copy. You can use `git checkout -b add-toogle-sort configure_by_zstyle` to create a new branch based on #46, which is still under develop and may change in the future.
2. I agree with romkatv. `toggle-sort-fzf-tab` is not a necessary feature, it won't be frequently used by most users and it can be implemented easily by users themselves.

File: fzf-tab.zsh (hunk 1 above)
Comment (Aloxaf) [reply fzf-tab.zsh]: Does `zstyle ':fzf_tab:*:options' sort false` work as your expect?
图片

**Comment by webmastak** [comment]
I will leave her for myself.

File: fzf-tab.zsh (hunk 1 above)
Comment (Aloxaf) [reply fzf-tab.zsh]: Oh I introduced a bug again, fixed in https://github.com/Aloxaf/fzf-tab/pull/46/commits/d2a8074f63a621e13b7e4f525c746448be71d589

Comment (Aloxaf) [reply fzf-tab.zsh]: Oh I introduced a bug again, fixed in https://github.com/Aloxaf/fzf-tab/pull/46/commits/d2a8074f63a621e13b7e4f525c746448be71d589.
I need to write some tests...

**Pull request closed without merging**